docs/getting-started/v3-migration.md

```diff
@@ -99,14 +180,25 @@ Animation system was completely rewritten in Chart.js v3. Each property can now
 * `Chart.radiusLength`
 * `Chart.types`
 * `Chart.Tooltip` is now provided by the tooltip plugin. The positioners can be accessed from `tooltipPlugin.positioners`
+* `ILayoutItem.minSize`
+
+#### Dataset Controller
```

Comment:
I kind of think it was easier to read without the sub-headings

Reply:
Open to removing them, I just felt they broke up the long list